Software/Development/SemesterProject/System/scheduler.c

```diff
@@ -76,8 +76,7 @@ void runTask(void (*functionPTR)())
  */
 void task100us(void)
 {//running at switching 10kHz (update with switching frequency);
-    setFrequencyFromPot();
-
+    openLoopVFControl();
```

Comment:
the task is called from controlTask, not from the scheduler

Reply:
Task for Nicolai!

Reply:
I am calling it in controlTask, but i cannot delete it from the scheduler